website/src/content/deployment-examples/render.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Deploying to Render
+---
+
+This deployment example refers to [Render](https://render.com/).
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [Render](https://app.netlify.com) account.
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, make sure to provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationId) that you got when you configured the Custom Application in the Merchant Center.
+
+Moreover, you need to provide the [production URL](/api-reference/application-config#envproductionurl) from your Netlify site. You can keep the standard Netlify URL `https://<site>.netlify.app` or provide your own custom domain.
```

Comment:
Here we're mentioning Netlify when these are the Render docs.

Reply:
Good catch!